editoast/src/views/infra/pathfinding.rs

```diff
@@ -177,14 +184,45 @@ fn compute_path(
     k: u8,
 ) -> Vec<PathfindingOutput> {
     let start = &input.starting;
-    let start = PathfindingStep::new_init(start.track.0.clone(), start.position, start.direction);
+    let start = PathfindingStep::new_init(start.track.0.clone(), start.position);
 
     let track_sections = infra_cache.track_sections();
     // Transform a length (in m) into a cost (in mm). This provide the Ord implementation for our cost using u64.
     let into_cost = |length: f64| (length * 100.).round() as u64;
     let get_length = |track: &String| track_sections[track].unwrap_track_section().length;
     let success = |step: &PathfindingStep| step.found;
+    let mut best_distance = u64::MAX;
```

Comment:
We should set this value to the geographical distance between the points multiplied by two.
Note: We can use the bounding boxes to calculate this.

Reply:
I added a commit doing so.